src/components/home/HomePage.js

```diff
@@ -26,20 +25,7 @@ function HomePage(props) {
         <div id="free-container" className="content-container">
           <Free />
         </div>
-        <div className="team-container">
-          <Team />
```

Comment:
You can delete team.js also if you want

Reply:
Nah, I intentionally didn't delete it, so we can potentially use it as a reference for later for the About page.